iocage_lib/ioc_json.py

```diff
@@ -2200,6 +2235,20 @@ def json_check_prop(self, key, value, conf):
         elif key in props.keys():
             # Either it contains what we expect, or it's a string.
 
+            if DEPRECATED in props[key]:
+                iocage_lib.ioc_common.logit(
+                    {
+                        'level': 'EXCEPTION',
+                        'message': f'The `{key}` parameter is deprecated, is '
+                                   'no longer in use by iocage, and cannot '
```

Comment:
*`iocage. It cannot be set`

Reply:
Dang. I see the intent of your above comment here. Resolving.